plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/exec/DBCDriverException.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.model.exec;
+
+import org.jkiss.dbeaver.model.connection.DBPDriver;
+
+public class DBCDriverException extends DBCException {
+    private static final long serialVersionUID = 1L;
```

Comment:
We don't serialize objects

Reply:
We are, pls check the inheritance of DBException.

Reply:
It's present here for some legacy reasons without any real usage.